systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlST.java

```diff
@@ -79,9 +81,28 @@ void testAutoCreationOfCruiseControlTopics(ExtensionContext extensionContext) {
                 .editKafka()
                     .addToConfig("auto.create.topics.enable", "false")
                 .endKafka()
+                .editCruiseControl()
+                    .withResources(new ResourceRequirementsBuilder()
+                        .addToLimits("memory", new Quantity("256Mi"))
+                        .addToLimits("cpu", new Quantity("1"))
+                        .addToRequests("memory", new Quantity("256Mi"))
+                        .addToRequests("cpu", new Quantity("0.8"))
+                        .build())
```

Comment:
Maybe leaving out the CPU completely would be better -> lower chance of conflict or of running out of resources.

Reply:
I will leave there 0.1 CPU for both, as we don't have such test for CC. Would you approve it with that? Else I would need to duplicate code from `assertResources()` and also possibly write new test as well for that CPU checks :-)

Reply:
Update: Will remove CPU, as CC is not able to start with 0.1 CPU.